src/pages/PullRequestPage/PullCoverage/routes/IndirectChangesTab/IndirectChangedFiles/hooks/useIndirectChangedFilesTable.ts

```diff
@@ -0,0 +1,165 @@
+import { SortingState } from '@tanstack/react-table'
```

Comment:
Thoughts on adding some function header comments detailing what each of these hooks is meant for at a high level?

I will admit it's a bit of a selfish ask but I think it'd help other newcomers of the codebase down the line too!

Reply:
I will keep this in mind for more abstract hooks. I think in this case, it seems fairly self-evident that it sends data to populate the indirect files changed table.

Do you think that is the case? Just to eliminate any curse of knowledge biases.

Reply:
Looking at the directory structure I think in this case it's pretty evident